packages/ui-components/test/unit/components/UIGuidedAnswerBox.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { Callout } from '@fluentui/react';
+import type { ICalloutContentStyles, IStyle } from '@fluentui/react';
+import * as Enzyme from 'enzyme';
```

Comment:
Please use [react testing library](https://testing-library.com/docs/react-testing-library/intro/) instead of enzyme as it appears to no longer be maintained and does not support newer react versions.

Reply:
I used what is already in use in the other tests, was that wrong?